pom.xml

```diff
@@ -28,6 +28,7 @@
 		<dependency>
 			<groupId>net.imglib2</groupId>
 			<artifactId>imglib2</artifactId>
+			<version>2.8.1-SNAPSHOT</version>
```

Comment:
We cannot merge that until there is a non-SNAPSHOT release of ImgLib2 or @ctrueden will kill you.  @tpietzsch @ctueden @StephanPreibisch are you ok with releasing imglib2-2.9.0 quickly?  This release should contain the `RandomAccessiblePair` and @tpietzsch's `Views` fix https://github.com/imglib/imglib2/pull/129.